pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "poetry-demo"
+version = "0.1.0"
+description = ""
+authors = ["Tone Vanden Bergh <vdbtone@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+pytest = "^6.1.2"
+invoke = "^1.4.1"
```

Comment:
is deze nodig?

Reply:
Als je poetry init doet op een lege repo maakt hij zelf deze file aan.
Ik heb gwn wat dependacys toegevoegd kwestie van wat voorbeelden te hebben. 
Toughts?